morphia/src/test/java/org/mongodb/morphia/query/TestQuery.java

```diff
@@ -1172,7 +1173,9 @@ public void run() {
             }
         }, 0, 500, TimeUnit.MILLISECONDS);
 
-        final Iterator<CappedPic> tail = query.tail();
+        final Iterator<CappedPic> tail = query
```

Comment:
It's a change I made to make sure when i update the docs that I do so with code that's correct.